atf_eregs/settings/prod.py

```diff
@@ -31,7 +31,11 @@
     HAYSTACK_CONNECTIONS['default'] = {
         'ENGINE': ('haystack.backends.elasticsearch_backend.'
                    'ElasticsearchSearchEngine'),
-        'URL': elastic_service.credentials['uri'],
+        'URL': 'http://{}:{}@{}:{}'.format(
```

Comment:
I just tried it out, the URI includes `https` and it fails :/

I checked the protocol on E/W and it was `http`. Of course, we want https though.

Part of the log dump below:

```
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR Failed to clear Elasticsearch index: ConnectionError(EOF occurred in violation of protocol (_ssl.c:590)) caused by: SSLError(EOF occurred in violation of protocol (_ssl.c:590))
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR Traceback (most recent call last):
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/backends/elasticsearch_backend.py", line 234, in clear
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR     self.conn.indices.delete(index=self.index_name, ignore=404)
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/client/utils.py", line 69, in _wrapped
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR     return func(*args, params=params, **kwargs)
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/client/indices.py", line 198, in delete
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR     params=params)
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/transport.py", line 307, in perform_request
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR     status, headers, data = connection.perform_request(method, url, params, body, ignore=ignore, timeout=timeout)
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/connection/http_urllib3.py", line 83, in perform_request
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR     raise SSLError('N/A', str(e), e)
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]ERR SSLError: ConnectionError(EOF occurred in violation of protocol (_ssl.c:590)) caused by: SSLError(EOF occurred in violation of protocol (_ssl.c:590))
2016-11-10T12:47:04.49-0500 [APP/PROC/WEB/0]OUT All documents removed.
2016-11-10T12:47:04.51-0500 [APP/PROC/WEB/0]OUT Indexing 15089 documents
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR ERROR:root:Error updating regcore using default
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR Traceback (most recent call last):
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 188, in handle_label
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     self.update_backend(label, using)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 233, in update_backend
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     do_update(backend, index, qs, start, end, total, verbosity=self.verbosity, commit=self.commit)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 96, in do_update
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     backend.update(index, current_qs, commit=commit)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/backends/elasticsearch_backend.py", line 193, in update
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     bulk(self.conn, prepped_docs, index=self.index_name, doc_type='modelresult')
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 188, in bulk
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     for ok, item in streaming_bulk(client, actions, **kwargs):
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 160, in streaming_bulk
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     for result in _process_bulk_chunk(client, bulk_actions, raise_on_exception, raise_on_error, **kwargs):
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 89, in _process_bulk_chunk
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     raise e
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR SSLError: ConnectionError(EOF occurred in violation of protocol (_ssl.c:590)) caused by: SSLError(EOF occurred in violation of protocol (_ssl.c:590))
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR Traceback (most recent call last):
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "manage.py", line 10, in <module>
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     execute_from_command_line(sys.argv)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     utility.execute()
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     self.fetch_command(subcommand).run_from_argv(self.argv)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     self.execute(*args, **cmd_options)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     output = self.handle(*args, **options)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/home/vcap/app/atf_eregs/management/commands/refresh.py", line 15, in handle
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     interactive=False, remove=True)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/__init__.py", line 120, in call_command
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     return command.execute(*args, **defaults)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     output = self.handle(*args, **options)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/rebuild_index.py", line 26, in handle
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     call_command('update_index', **options)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/__init__.py", line 120, in call_command
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     return command.execute(*args, **defaults)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     output = self.handle(*args, **options)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 183, in handle
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR     return super(Command, self).handle(*items, **options)
2016-11-10T12:47:04.88-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/django/core/management/base.py", line 623, in handle
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     label_output = self.handle_label(label, **options)
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 188, in handle_label
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     self.update_backend(label, using)
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 233, in update_backend
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     do_update(backend, index, qs, start, end, total, verbosity=self.verbosity, commit=self.commit)
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/management/commands/update_index.py", line 96, in do_update
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     backend.update(index, current_qs, commit=commit)
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/haystack/backends/elasticsearch_backend.py", line 193, in update
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     bulk(self.conn, prepped_docs, index=self.index_name, doc_type='modelresult')
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 188, in bulk
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     for ok, item in streaming_bulk(client, actions, **kwargs):
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 160, in streaming_bulk
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     for result in _process_bulk_chunk(client, bulk_actions, raise_on_exception, raise_on_error, **kwargs):
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR   File "/app/.heroku/python/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 89, in _process_bulk_chunk
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR     raise e
2016-11-10T12:47:04.89-0500 [APP/PROC/WEB/0]ERR elasticsearch.exceptions.SSLError: ConnectionError(EOF occurred in violation of protocol (_ssl.c:590)) caused by: SSLError(EOF occurred in violation of protocol (_ssl.c:590))
```